app/core/Engine.js

```diff
@@ -66,8 +66,9 @@ class Engine {
 				{
 					ipfsGateway: AppConstants.IPFS_DEFAULT_GATEWAY_URL,
 					useStaticTokenList:
-						(initialState.PreferencesController && initialState.PreferencesController.useStaticTokenList) ||
-						true,
+						initialState?.PreferencesController?.useStaticTokenList === undefined
+							? true
+							: initialState.PreferencesController.useStaticTokenList,
```

Comment:
Why you didn't this notation?

```suggestion
						initialState?.PreferencesController?.useStaticTokenList === undefined
						|| initialState.PreferencesController.useStaticTokenList,
```